src/plugins/intel_gpu/src/plugin/ops/non_max_suppression.cpp

```diff
@@ -14,6 +14,61 @@
 #include "intel_gpu/runtime/debug_configuration.hpp"
 
 namespace ov::intel_gpu {
+namespace {
+std::vector<cldnn::input_info> reorder_nms_inputs_if_needed(
```

Comment:
Please make these functions as `static` so that it does not conflict with other parts of software. (and for other new functions as well)

Reply:
Updated it! Thanks!